community/datadog/templates/secrets.yaml

```diff
@@ -0,0 +1,38 @@
+# API Key
```

Comment:
Creating secrets in this way is insecure in Helm 2; you should provide an alternative, either through documenting a secure approach in the README or through a script provided in the chart.

Reply:
We have a [section](https://github.com/IBM/charts/pull/690/files#diff-4a69cecce308bd45df04b0a6a837cecbR43) in the `README.md` that explains how to create first a secret. and the pass the secret name to the chart install command.
 
We also propose other options in or docs: https://docs.datadoghq.com/agent/kubernetes/?tab=helm#using-secret-files

Reply:
GitHub

Reply:
Github